### DISCUSSION TITLE
Can't find BitmapData asset with ID even that it is there

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2595] [Q&A]
this is the code for initiating the image, with character variable being gf
(note: i did try the same code but with "true" in loadGraphic, that var defines if image is animated)
![image](https://user-images.githubusercontent.com/60806448/177029981-1e078b9f-ca4c-4b4c-8b75-78c80ebe8d65.png)

so, the path will look like "assets/images/gf.png", right?
![image](https://user-images.githubusercontent.com/60806448/177030015-bb9effe7-34fd-4479-b62d-a42c270ada47.png)

then uhh.. why this happens? am I doing something wrong
![image](https://user-images.githubusercontent.com/60806448/177029970-5084e1c2-62a2-4d9c-9f56-05156d11d4e7.png)

**Comment by RichardBray** [comment]
I'm sure there's a simple solution to this. Would it be possible to look at where your code is kept in relation to your assets? And was the file always a png? Or was it a different format before and now has been renamed to a .png file?
